bom/runtime/pom.xml

```diff
@@ -28,6 +28,51 @@
                 <scope>import</scope>
             </dependency>
 
+            <!-- Resolve conflicts among platform participants -->
+            <dependency>
+                <groupId>io.debezium</groupId>
+                <artifactId>debezium-api</artifactId>
+                <version>${debezium.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.debezium</groupId>
+                <artifactId>debezium-connector-mysql</artifactId>
+                <version>${debezium.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.debezium</groupId>
+                <artifactId>debezium-connector-postgres</artifactId>
+                <version>${debezium.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.debezium</groupId>
+                <artifactId>debezium-connector-sqlserver</artifactId>
+                <version>${debezium.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.debezium</groupId>
+                <artifactId>debezium-core</artifactId>
+                <version>${debezium.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.debezium</groupId>
+                <artifactId>debezium-dll-parser</artifactId>
+                <version>${debezium.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.debezium</groupId>
+                <artifactId>debezium-embedded</artifactId>
+                <version>${debezium.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.testcontainers</groupId>
+                <artifactId>testcontainers-bom</artifactId>
+                <type>pom</type>
+                <scope>import</scope>
+                <version>1.14.3</version>
+            </dependency>
```

Comment:
I'm not entirely sure this is a good idea to have it here given we decided to remove it from the Quarkus core BOM.

@aloubyansky what's your take on that one?

Reply:
BTW, additional question: is it a test only dependency?

Reply:
Whether it's only a test dep is the key question. Are they all necessary in the user-facing BOM @ppalaga ?

Reply:
Just to make sure: Have you guys seen this: https://github.com/quarkusio/quarkus/issues/9968#issuecomment-650835627
and the resulting PR #81 here in quarkus-platform?

Reply:
@famod thanks, I missed it and your analysis is indeed very useful. I'll try to come up with a contingency plan for 1.6 and a better plan for 1.7.